core/src/main/java/hudson/diagnosis/OldDataMonitor.java

```diff
@@ -458,5 +459,15 @@ public String getDescription() {
         public String getDisplayName() {
             return Messages.OldDataMonitor_DisplayName();
         }
+
+        @Override
+        public Permission getRequiredPermission() {
+            return Jenkins.MANAGE;
+        }
+    }
+
+    @Override
+    public Permission getRequiredPermission() {
+        return Jenkins.MANAGE;
```

Comment:
This change (and the monitor) is undesirable, since Manage does not imply global Item/Read, or even Item/Discover, so this might list items the user is not entitled to see.

Or it needs to be documented _very_ carefully.

Reply:
I am leaning toward moving this back to requiring `Jenkins.ADMINISTER`

Reply:
+1 for reverting

Reply:
reverted back to `Jenkins.ADMINISTER`